### PR TITLE
s22-6pm-1 Kevin - Added .DS_Store to gitignore file

### PR DESCRIPTION
**Pull request by notkevin12**
KL - Added .DS_Store to gitignore file to prevent further unintentional inclusion of .DS_Store to git commits. Doesn't close any issues.

## Before

image

## After 

image

### CODE REVIEW

**Review by anuraghs — APPROVED** [review]
LGTM

**Review by hanzemeng — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -49,4 +49,7 @@ build/
 .vscode/
 
 ### Local Configuration ###
-.env
\ No newline at end of file
+.env
+
+### apple sux ###
```
Comment (andrewhlu) [inline .gitignore:54]: true

**Review by andrewhlu — APPROVED** [review]
LGTM, thanks for this change!

**Review by pconrad — COMMENTED** [review]
LGTM

**Pull request merged**